packages/storage-driver-cloudinary/src/index.test.ts

```diff
@@ -761,17 +764,32 @@ describe('#move', () => {
 		expect(driver['fullPath']).toHaveBeenCalledWith(sample.path.dest);
 	});
 
-	test('Gets resource type for src', async () => {
+	test('Gets public id for src', async () => {
 		await driver.move(sample.path.src, sample.path.dest);
-		expect(driver['getResourceType']).toHaveBeenCalledWith(sample.path.srcFull);
+		expect(driver['getPublicId']).toHaveBeenCalledWith(sample.path.srcFull);
+	});
+
+	test('Gets public id for dest', async () => {
+		await driver.move(sample.path.dest, sample.path.dest);
+		expect(driver['getPublicId']).toHaveBeenCalledWith(sample.path.destFull);
+	});
+
+	test('Gets folder path for src', async () => {
+		await driver.move(sample.path.src, sample.path.src);
+		expect(driver['getFolderPath']).toHaveBeenCalledWith(sample.path.srcFull);
+	});
+
+	test('Gets folder path for dest', async () => {
+		await driver.move(sample.path.dest, sample.path.dest);
+		expect(driver['getFolderPath']).toHaveBeenCalledWith(sample.path.destFull);
 	});
 
 	test('Creates signature for body parameters', async () => {
-		await driver.move(sample.path.src, sample.path.dest);
+		await driver.move(sample.path.input, sample.path.input);
 
 		expect(driver['getFullSignature']).toHaveBeenCalledWith({
-			from_public_id: sample.publicId.src,
-			to_public_id: sample.publicId.dest,
+			from_public_id: joinActual(sample.path.inputFolder, sample.publicId.input),
+			to_public_id: joinActual(sample.path.inputFolder, sample.publicId.input),
```

Comment:
Could we still stick to `src` & `dest`? Seems a bit confusing to me otherwise. Or what's the idea of using `input` for both args in this test case?

Reply:
Yes, we can. I was trying to put the minimum changes possible but it turned out to be much unreadable.
Have made a new push with those changes 👍